docs/DeveloperGuide.md

```diff
@@ -827,8 +827,20 @@ testers are expected to do more *exploratory* testing.
 
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
+   
+1. Shutdown
 
-1. _{ more test cases …​ }_
+   1. Launch the jar file <br> 
+   Expected: Shows the GUI with a set of sample volunteers and events. The window size may not be optimum.
```

Comment:
suggestion: `Note that the window may not be optimised for your screen`

Reply:
oki